README.md

```diff
@@ -1,5 +1,39 @@
 # xeus-calc
 
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/QuantStack/xeus-calc/master)
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/QuantStack/xeus-calc/stable?filepath=notebooks/Xeus-calc.ipynb)
```

Comment:
You don't have a stable branch, do you? We use stable branches on other projects for relying on specific versions of our projects. Maybe you don't need this.

Reply:
If you try this url, you should see it failling

Reply:
I think they should keep the logical of stable and master branches. And therefore create a stable branch

Reply:
Yes, but not until they made a first release right? Otherwise, they would need to update the stable branch at each commit if they want an up-to-date binder

Reply:
true!

Reply:
Yes, we just have '' master " branch.